packages/lesswrong/components/tagging/TagsTooltip.tsx

```diff
@@ -54,74 +105,81 @@ const DefaultPreviewWrapper: TagsTooltipPreviewWrapper = ({children}) => (
   <>{children}</>
 );
 
-const styles = defineStyles("TagsTooltip", theme => ({
-  tooltip: isFriendlyUI
-    ? {}
-    : {
-      padding: 0,
-      background: theme.palette.panelBackground.default,
-      boxShadow: theme.palette.boxShadow.lwTagHoverOver,
-    },
-  tooltipTitle: isFriendlyUI
-    ? {}
-    : {
-      maxWidth: "unset",
-    },
-  loading: {
-    paddingLeft: 16,
-    paddingRight: 32,
-    paddingBottom: 24,
-  },
-  redLinkTooltip: {
-    paddingTop: 8,
-    paddingLeft: 16,
-    paddingRight: 16,
-    width: 500,
-    paddingBottom: 6,
-    [theme.breakpoints.down('xs')]: {
-      width: "100%",
-    }
-  },
-  redLinkTooltipTitle: {
-    fontSize: '13px',
-    marginBottom: 4,
-    color: theme.palette.text.secondary,
-  },
-}));
 
 const RedLinkTooltip = ({ tag, slug }: {
   tag: TagBasicInfo | null
   slug?: string
 }) => {
   const classes = useStyles(styles);
   const { Typography, ContentStyles, TagHoverPreview, Loading } = Components;
-  const { results: pingbacks, loading: loadingPingbacks } = useRedLinkPingbacks(tag?._id);
-  const title = inferRedLinkTitle(tag, slug??null);
-
-  return <div className={classes.redLinkTooltip}>
-    <Typography variant='title'>
-      {title}
-    </Typography>
-    <ContentStyles contentType='tag'>
-      This red link was used on {pingbacks ? pingbacks.length : <Loading/>} page{pingbacks ? (pingbacks.length > 1 ? "s" : "") : "(s)"}:
-      <ul>
-        {pingbacks?.map(pingback => <li key={pingback._id}>
-          <TagHoverPreview
-            targetLocation={{ params: { slug: pingback.slug }, hash: '', query: {} } as AnyBecauseTodo}
-            href={tagGetUrl({slug: pingback.slug})} noPrefetch
-          >
-            <Link to={tagGetUrl({slug: pingback.slug})}>
-              {pingback.name}
-            </Link>
-          </TagHoverPreview>
-        </li>)}
-      </ul>
-    </ContentStyles>
-    <ContentStyles contentType='tag' className={classes.redLinkTooltipTitle}>
-      A red link highlights author's intention to point at a concept that doesn't have a satisfactory explanation written (by their standards).
-    </ContentStyles>
-  </div>
-}
+  const { tag: currentlyViewingTag } = useTagPageContext() ?? {};
+  const excludedPingbackTagIds = currentlyViewingTag?._id ? [currentlyViewingTag._id] : undefined;
+  const { results: pingbacks, loading: loadingPingbacks } = useRedLinkPingbacks(tag?._id, excludedPingbackTagIds);
```

Comment:
`useRedLinkPingbacks` should be refactored to include another `useMulti` on multidocuments to fetch pingbacks coming from lenses, and multidocuments needs a corresponding view

Reply:
(and then `useRedLinkPingbacks` should _probably_ combine the two results & loading states before returning them?)